### PR TITLE
Add DiscountRate unit type [delivers #157270338]

### PR DESCRIPTION
**Pull request by jim**
## Description

In the codebase we are inconsistent with how we represent discount rates. In some places the code expects a value between `0` and `1`, and in others, it assumes a value between `0` and `100`. This PR standardizes on the former and adds a new type, `unit.DiscountRate`, so that the compiler will help us work with these values properly.

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157270338) for this change

### CODE REVIEW

**Comment by macrael** [comment]
I'm not sure what you mean by "Addressed the representation issue enough." We have a method called Int() on Cents, so it would make sense to me that a method called Float64() would serve the same purpose here. But It's not that important to me, I expect neither to be called frequently in code.

**Comment by jim** [comment]
@macrael I think I am overthinking it. I was concerned that a datatype of `float64` doesn't communicate/indicate how the value is represented (see above for the two options). But like you mentioned, we just have an `Int` method in `Cents` and that seems to be fine. So I'll change this PR to use `Float64` instead of `Decimal`.

File: pkg/handlers/personally_procured_move_sit_estimate.go [hunk 1]
```diff
@@ -69,10 +69,10 @@ func (h ShowPPMSitEstimateHandler) Handle(params ppmop.ShowPPMSitEstimateParams)
 		return responseForError(h.logger, err)
 	}
 
-	// TODO (rebecca): Settle how percentages are handled
-	inverseDiscount := (100.00 - sitDiscount) / 100
+	inverseDiscount := sitDiscount.Invert()
```
Comment (akostibas) [inline pkg/handlers/personally_procured_move_sit_estimate.go]: I'm a little late to the party, but what do you think about making a function `sitDiscount.ApplyToSum(sitTotal)`? Then we don't have to fuss with the "inverse" / "invert" stuff, and we know it's done properly.

Comment (jim) [reply pkg/handlers/personally_procured_move_sit_estimate.go]: I like this idea and will implement a version of it.

File: pkg/testdatagen/make_tsp_performance_records.go [hunk 2]
```diff
@@ -94,8 +97,8 @@ func MakeTSPPerformanceData(db *pop.Connection, rounds string) {
 			&qualityBand,
 			bvs,
 			offers,
-			discountRate,
-			discountRate,
+			lhRate,
+			lhRate,
```
Comment (kilbergr) [inline pkg/testdatagen/make_tsp_performance_records.go]: I see why you've done this way, but it's a little misleading to call these both lhRate s. Not a blocking request.

**Pull request merged**